posthog/api/utils.py

```diff
@@ -238,10 +238,11 @@ def check_definition_ids_inclusion_field_sql(
 
 
 # keep in sync with posthog/plugin-server/src/utils/db/utils.ts::safeClickhouseString
-def safe_clickhouse_string(s: str) -> str:
+def safe_clickhouse_string(s: str, with_counter=True) -> str:
     matches = SURROGATE_REGEX.findall(s or "")
     for match in matches:
-        SURROGATES_SUBSTITUTED_COUNTER.inc()
+        if with_counter:
+            SURROGATES_SUBSTITUTED_COUNTER.inc()
```

Comment:
someone is measuring data as it passes through here, let's not pollute their graphs by suddenly firing lots more data through it 🙈